src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -1954,9 +1978,13 @@ public Vector4 WorldColor
             get
             {
                 Vector4 temp = new Vector4(0.0f, 0.0f, 0.0f, 0.0f);
+#if NUI_PROPERTY_CHANGE_3
+                Object.InternalRetrievingPropertyVector4(SwigCPtr, View.Property.WorldColor, temp.SwigCPtr);
```

Comment:
same as above

Reply:
I found that `WorldColor` is exactly equal with `CurrentWorldColor` intenrally. So I make it `ViewInternal.cs` API, which we already changed.